logger/medialogutils/cmd.go

```diff
@@ -58,8 +64,8 @@ func (l *HandlerLogger) Write(p []byte) (n int, err error) {
 		case strings.HasPrefix(line, "0:00:"):
 			// ignore cuda and template not mapped gstreamer warnings
 			continue
-		case strings.HasPrefix(line, "turnc"):
-			// warn on turnc error
+		case len(line) > 6 && downgrade[line[:6]]:
```

Comment:
Case insensitive check?

Reply:
case sensitive is fine, they're always the same logs